dapps/faucet/dapp.go

```diff
@@ -124,11 +124,11 @@ func run(*node.Plugin) {
 }
 
 func configureEvents() {
-	messagelayer.Tangle().Events.MessageSolid.Attach(events.NewClosure(func(cachedMessage *message.CachedMessage, cachedMessageMetadata *tangle.CachedMessageMetadata) {
-		defer cachedMessage.Release()
-		defer cachedMessageMetadata.Release()
+	messagelayer.Tangle().Events.MessageSolid.Attach(events.NewClosure(func(cachedMessage *tangle.CachedMessageEvent) {
```

Comment:
I think it is a bit odd to call it `cachedMessage` if it is of type `*CachedMessageEvent`, that has a `cachedMessage` and a `cachedMessageMetadata` field. I would rather call it `cachedMessageEvent`, `eventStruct`, `eventParameter` or any combination of these to signal that it is a struct only for this particular event.